### ISSUE TITLE
Minor workflow ui tweaks

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#14155]
- [x] make edit schema dialog centered
- [x] show delete icon for archived schemes
- [x] crumbtrail when editing steps - schemes --> scheme
- [x] Archiving and deleting workflows do not refresh the screen

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/14156

**Comment by fabrizzio-dotCMS** [comment]
I found there were problems refreshing the UI on the Save, Copy, Archive and Delete Actions
I initially got them all fixed. Then I found a problem with the refresh after the delete action is executed.
 
Workflows are removed asynchronously causing that for a brief period they will still appear on the screen while the deletion process completes. 
We agreed that handling the refresh of the UI for the delete action will be done through a WebSocket event. for that purpose, a separate ticket will be created
